.changes/unreleased/Dependencies-20221010-181501.yaml

```
@@ -0,0 +1,7 @@
+kind: Dependencies
+body: Bump google-cloud-bigquery dependency to < 3.3.3 ([#332](https://github.com/dbt-labs/dbt-bigquery/pull/332)
```

Embarrassing that I missed this previously -- this should just be:
```suggestion
body: Bump google-cloud-bigquery dependency to < 3.3.3
```

Reason:
When we generate the actual changelog, it will use the "Issue" and "PR" fields to create the links, so we don't want/need links in the "body" field.

---

Not a problem Doug. Thank you for reviewing